src/jdk.jdwp.agent/share/native/libjdwp/eventHelper.c

```diff
@@ -90,6 +100,13 @@ typedef struct ReportVMInitCommand {
     jthread thread;
 } ReportVMInitCommand;
 
+#if defined(J9VM_OPT_CRIU_SUPPORT)
+typedef struct ReportVMRestoreCommand {
+    jbyte suspendPolicy; /* NOTE: Must be the first field */
+    jthread thread;
+} ReportVMRestoreCommand;
+#endif /* defined(J9VM_OPT_CRIU_SUPPORT) */
```

Comment:
Removed `ReportVMRestoreCommand`, and reuse `ReportVMInitCommand` instead.

Reply:
Part of the motivation for not defining a new type was to avoid adding a new member to the `union` in `HelperCommand`. Please just use `reportVMInit` instead of defining `reportVMRestore`.